helpers/auth/authentication.py

```diff
@@ -63,10 +64,39 @@ def decode_token(self):
                 'Invalid token. Please Provide a valid token!'
             }), 401
 
-    def save_user(self):
+    def get_user_details_from_api(self, email, *expected_args):
```

Comment:
This function is called only on line `117`  and `expected_args` is not being passed. Why is it an argument here?

Reply:
@rajeman  `expected_args` is used as an argument in the `handle_http_error` function on line 86,93,95